interface/reports/patient_list_creation.php

```diff
@@ -246,582 +319,773 @@ function srch_option_change(elem) {
+                    <tr>
+                        <td class="text"><strong><?php echo xlt('Total Number of Patients')?>:</strong>&nbsp;<span id="total_patients"><?php echo text(count(array_unique($patient_arr))); ?></span></td>
+                    </tr>
+                </table>
 
-            if (!empty($_POST['sortby']) && !empty($_POST['sortorder'])) {
-                if ($_POST['sortby'] == "communications") {
-                    $odrstmt = "ORDER BY ROUND((LENGTH(communications) - LENGTH(REPLACE(communications, ',', '')))/LENGTH(',')) " . escape_sort_order($_POST['sortorder']) . ", communications " . escape_sort_order($_POST['sortorder']);
-                } elseif ($_POST['sortby'] == "insurance_companies") {
-                    $odrstmt = "ORDER BY ins_provider " . escape_sort_order($_POST['sortorder']);
+                <table class='table' width='90%' align="center" cellpadding="5" cellspacing="0" style="font-family: Tahoma;" border="0">
+                    <tr class="bg-light" style="font-size:15px;">
+        <?php }
+        foreach (array_keys($report_options_arr[$srch_option]["cols"]) as $report_col_key => $report_col) {
+            if (!$csv) {
+                echo '<td ';
+                if (isset($report_options_arr[$srch_option]["cols"][$report_col]["width"])) {
+                    $width = $report_options_arr[$srch_option]["cols"][$report_col]["width"];
+                    if (str_contains($width, '%')) {
+                        echo 'width="' . $width . '" ';
+                    } else if ($width == 'nowrap') {
+                        echo 'width="1%" style="white-space: nowrap;" ';
+                    } else {
+                        echo 'colspan="' . $width . '" ';
+                    }
+                }
+                echo 'class="font-weight-bold">' . xlt($report_options_arr[$srch_option]["cols"][$report_col]["heading"]);
+                if (isset($report_options_arr[$srch_option]["sort_cols"])) {
+                    if ($report_options_arr[$srch_option]["sort_cols"] != 0) {
+                        if (
+                            ($report_options_arr[$srch_option]["sort_cols"] > 0 && $report_col_key < $report_options_arr[$srch_option]["sort_cols"])
+                            || ($report_options_arr[$srch_option]["sort_cols"] < 0 && $report_col_key < $report_options_arr[$srch_option]["sort_cols"] + count($report_options_arr[$srch_option]["cols"]))
+                        ) {
+                            echo $sortlink[$report_col_key];
+                        }
+                    }
                 } else {
-                    $odrstmt = "ORDER BY " . escape_identifier($_POST['sortby'], $sort, true) . " " . escape_sort_order($_POST['sortorder']);
+                    echo $sortlink[$report_col_key];
+                }
+                echo '</td>';
+            } else {
+                echo csvEscape(xlt($report_options_arr[$srch_option]["cols"][$report_col]["heading"]));
```

Comment:
since this is going to csv, no need to escape with xlt and no need to translate since will be translated above